pkg/runtime/runtime.go

```diff
@@ -1976,7 +1987,7 @@ func (a *DaprRuntime) publishMessageHTTP(ctx context.Context, msg *pubsubSubscri
 	// Every error from now on is a retriable error.
 	log.Warnf("retriable error returned from app while processing pub/sub event %v, topic: %v, body: %s. status code returned: %v", cloudEvent[pubsub.IDField], cloudEvent[pubsub.TopicField], body, statusCode)
 	diag.DefaultComponentMonitoring.PubsubIngressEvent(ctx, msg.pubsub, strings.ToLower(string(pubsub.Retry)), msg.topic, elapsed)
-	return errors.Errorf("retriable error returned from app while processing pub/sub event %v, topic: %v, body: %s. status code returned: %v", cloudEvent[pubsub.IDField], cloudEvent[pubsub.TopicField], body, statusCode)
+	return fmt.Errorf("retriable error returned from app while processing pub/sub event %v, topic: %v, body: %s. status code returned: %v, err: %w", cloudEvent[pubsub.IDField], cloudEvent[pubsub.TopicField], body, statusCode, ErrRetriable)
 }
 
 func (a *DaprRuntime) publishMessageGRPC(ctx context.Context, msg *pubsubSubscribedMessage) error {
```

Comment:
Not sure if this method needs to be modified at the same time

Reply:
@artursouza  Do we also need to distinguish between retriable and non-retriable errors in the `publishMessageGRPC` func?